website/docs/system.md

```diff
@@ -23,6 +24,8 @@ If no system is specified, Premake will identify and target the current operatin
 * windows
 * xbox360
 
+To note: `emscripten` at the moment is only supported for the `gmake` and `gmake2` actions.
```

Comment:
For uniformity with the rest of the docs, use a separate "Availability" header and make any notes beneath that. See https://raw.githubusercontent.com/premake/premake-core/refs/heads/master/website/docs/debugformat.md

Reply:
Pretty sure that others values are not cross-generator neither.

Reply:
We should probably clean them up at some point, then.